docs/variables/setgame.md

```diff
@@ -0,0 +1,33 @@
+---
+id: setgame
+---
+
+# $(setgame)
+
+Changes the currently played game on stream.
+
+#### Parameters
+
+This variable takes **one** *required* parameter that is a **game** of what to set the stream directory to.
```

Comment:
This isn't true for either variable, it supports arguments after index 1 in the message too by default, ie `!setgame gta` with a command response of `$(setgame)` would work (for backwards compatibility with v1)

Reply:
mb, I've run it via eval and forgot it won't take the parameter that way 🙃